apps/website/src/server/notifications.ts

```diff
@@ -209,15 +210,12 @@ async function createDiscordNotifications({
 
   const tasks = [];
   for (const webhookUrl of webhookUrls) {
-    const relativeNotificationUrl = `/notifications/${id}`;
-    const link =
-      linkUrl ||
-      new URL(relativeNotificationUrl, env.NEXT_PUBLIC_BASE_URL).toString();
-    const content = `${title}\n${message}\n${link}`;
-
     tasks.push(
       triggerDiscordChannelWebhook({
-        content,
+        contentTitle: title || undefined,
```

Comment:
```suggestion
        contentTitle: title ?? undefined,
```
So we don't cast a falsey string to undefined